bolt-modules/ctrl/spec/functions/ctrl/do_until_spec.rb

```diff
@@ -31,4 +31,15 @@
     end.and_return("truthy"))
     expect(seq.length).to eq(1)
   end
+
+  it 'sleeps with an interval' do
+    now = Time.now
+
+    is_expected.to(run.with_params('interval' => 1, 'limit' => 3).with_lambda do
+      seq.pop
+    end.and_return('truthy'))
+
+    expect(Time.now - now).to be >= 1
```

Comment:
Maybe stub sleep and check that it's called twice? I would like to know that it's called after each invocation. Also it would be good to test that it's _not_ called when the first item is truthy  (ie. it doesn't sleep _unless_ it has to repeat the loop)